enterprise/frontend/src/embedding-sdk/README.md

```diff
@@ -285,25 +285,42 @@ const theme = {
     /** Color used for aggregations and breakouts context */
     summarize: "#88BF4D",
 
+    /** Color used to indicate successful actions and positive values/trends */
+    positive: "#BADC58",
+
+    /** Color used to indicate dangerous actions and negative values/trends */
+    negative: "#FF7979",
+
     /** Color used for popover shadows */
     shadow: "rgba(0,0,0,0.08)",
   },
 
-  table: {
-    cell: {
-      // Text color of cells, defaults to `text-dark`
-      textColor: "#4C5773",
+  components: {
```

Comment:
noticed that the structure in the current README is wrong, it should be `theme.components.table` not `theme.table`